### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by asolntsev**

### CODE REVIEW

File: gradlew.bat [hunk 1]
```diff
@@ -64,29 +64,14 @@ echo location of your Java installation.
 
 goto fail
 
-:init
```
Comment (ericbeland) [inline gradlew.bat:67]: Have these changes been tested on Windows? I don't have a Windows machine handy. Can you provide a rationale for the gradlew.bat changes so I can understand better why we should remove this?

**Review by ericbeland — COMMENTED** [review]
Looks ok except for my hesitation around the windows changes to gradlew. They might also be fine, but before I sign off, I want to understand the rationale.

File: gradlew.bat (hunk 1 above)
Comment (asolntsev) [reply gradlew.bat:67]: Hi. 
This file is automatically updated when you run command
```
./gradlew wrapper --gradle-version=6.7.1 --distribution-type=bin
```

It's the official [recommended way](https://gradle.org/install/#with-the-gradle-wrapper) to upgrade gradle.

No, I personally haven't tried to run this file on Windows, but I don't see any reasons why it should not work.

**Comment by asolntsev** [comment]
@ericbeland I have resolved conflicts and rebase the branch on latest `master` branch.

File: gradlew.bat (hunk 1 above)
Comment (ericbeland) [reply gradlew.bat:67]: Perfect, thanks for the explanation!

**Comment by ericbeland** [comment]
I'm ready to merge this, but can we pull out the minor bump on awaitility in this PR?  We have a PR here https://github.com/browserup/browserup-proxy/pull/270 generated by dependabot that does the awaitility bump, and that minor bump keeps failing to build (3 times), so I was planning to revisit that later.

**Comment by asolntsev** [comment]
@ericbeland Sorry, I don't understand what you are asking for. 
1. PR #270 upgrades `org.awaitility:awaitility` 4.0.2 -> 4.0.3
2. But current PR #317 already contains `org.awaitility:awaitility:4.0.3`. Nothing to pull here.

File: browserup-proxy-core/build.gradle [hunk 2]
```diff
@@ -58,19 +58,19 @@ uploadArchives {
...
+    api("xyz.rogfam:littleproxy:${littleProxyVersion}") {
         exclude(group: 'io.netty', module: 'netty-all')
     }
 
     implementation project(':browserup-proxy-mitm')
 
     implementation 'javax.xml.bind:jaxb-api:2.3.1'
 
-    implementation 'org.awaitility:awaitility:4.0.2'
+    implementation 'org.awaitility:awaitility:4.0.3'
```
Comment (ericbeland) [inline browserup-proxy-core/build.gradle:69]: When dependabot generated a PR for awaitiity 4.0.3, I could not for the life of me get the build tests to pass. I closed it out. I haven't done a dive into why our tests consistently failed, but until I do, I would be nervous to merge a PR with 4.0.3 in it.

**Review by ericbeland — APPROVED** [review]

**Pull request merged**